pages/examples/tables/sticky-headers/_examples/table-with-fixed-headers/README.md

```diff
@@ -3,19 +3,25 @@ title: "Table with fixed headers"
 compatibility:
   Keyboard only:
     status: pass
-    date: 2018-04-17
+    date: 2023-10-19
   NVDA:
```

Comment:
@NicSR, could you please ask a test expert to have a look at the new demo? I have successfully tested keyboard accessibility.

https://deploy-preview-422--accessibility-developer-guide.netlify.app/examples/tables/sticky-headers/_examples/table-with-fixed-headers/

Reply:
our test was successfuly aswell. Well done an thanks.